legacy/web/l10n/pt_BR.json

```diff
@@ -247,7 +247,7 @@
     "linkGoogleMaps": "Google Maps",
     "linkAppleMaps": "Apple Maps",
     "linkStreetView": "Street View",
-    "positionFixTime": "Definir tempo",
+    "positionFixTime": "Hora",
```

Comment:
This doesn't look like the right translation.

Reply:
I just put hour because I can't find an equivalent word to fix in this case. Maybe "Hora da posição" which means Position time.

I never heard of transifex, I'll check it out.

Reply:
There are 3 time parameters in Traccar. Each one is slightly different, so we need to make sure they are consistent and reflect what they actually mean.

Reply:
I tried to translate it to "Hora GPS" in transifex, don't know if anything happened.

Reply:
We sync it up every release.